### PR TITLE
(major rework) Removed all instances of Engineer Suite 2

### PR DESCRIPTION
**Pull request by EngineerSuite2**

### CODE REVIEW

File: FantasyModuleParser/FMP.csproj [hunk 1]
```diff
@@ -22,7 +22,7 @@
     <NuGetPackageImportStamp>
     </NuGetPackageImportStamp>
     <IsWebBootstrapper>false</IsWebBootstrapper>
-    <PublishUrl>publish\</PublishUrl>
+    <PublishUrl>C:\Users\Rachel Story\Desktop\Engineer Suite\Release Versions\</PublishUrl>
```
Comment (lorddusk) [inline FantasyModuleParser/FMP.csproj:25]: change this back to ``publish\``

File: FantasyModuleParser/Main/Views/About.xaml [hunk 2]
```diff
@@ -17,7 +17,7 @@
                     </GeometryGroup>
                 </Path.Data>
             </Path>
-            <Label Content="Written by Theodore Story, Kertic, and LordDusk" Padding="0" HorizontalContentAlignment="Center"/>
+            <Label Content="Written by Theodore Story, Darkpool, and Dusk" Padding="0" HorizontalContentAlignment="Center"/>
```
Comment (lorddusk) [inline FantasyModuleParser/Main/Views/About.xaml:20]: LordDusk, I just use Dusk for ease of talking in chats

File: FantasyModuleParser/Main/Views/Settings.xaml [hunk 3]
```diff
@@ -113,13 +113,14 @@
                             </StackPanel>
                         </StackPanel>
                         <StackPanel Orientation="Vertical" Margin="20,0,0,0">
-                            <Label Content="Engineer Suite Settings" Foreground="Gray" HorizontalAlignment="Left" Margin="10,20,0,0" VerticalAlignment="Top" FontWeight="Bold"/>
+                            <Label Content="Fantasy Module Paraser Settings" Foreground="Gray" HorizontalAlignment="Left" Margin="10,20,0,0" VerticalAlignment="Top" FontWeight="Bold"/>
```
Comment (lorddusk) [inline FantasyModuleParser/Main/Views/Settings.xaml:116]: Parser

File: FantasyModuleParser/NPC/Controllers/ActionController.cs [hunk 4]
```diff
@@ -114,5 +114,62 @@ public void LowerActionInNPCActionsList(ActionModelBase action)
...
+
+        public void RaiseActionInLairActionList(LairAction action)
+        {
+            ObservableCollection<LairAction> lairActions = GetNPCModel().LairActions;
+
+            int actionIndex = lairActions.IndexOf(action);
+            if (actionIndex == 0)
+            {
+                //Do nothing!  Already at the top of the list
+            }
```
Comment (lorddusk) [inline FantasyModuleParser/NPC/Controllers/ActionController.cs:126]: can be removed.

File: FantasyModuleParser/NPC/Models/Action/Enums/WeaponImmunity.cs [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+	{
+		[Description("No Special Weapon Immunity")]
+		NoSpecial = 0,
+		[Description("Immune to nonmagical Weapons")]
+		Nonmagical = 1,
+		[Description("Immune to nonmagical weapons that aren't silvered")]
+		NonmagicalSilvered = 2,
+		[Description("Immune to nonmagical weapons that aren't adamantine")]
+		NonmagicalAdamantine = 3,
+		[Description("Immune to nonmagical weapons that aren't cold-forged iron")]
```
Comment (lorddusk) [inline FantasyModuleParser/NPC/Models/Action/Enums/WeaponImmunity.cs:15]: Consistency error

File: FantasyModuleParser/NPC/Models/Action/Enums/WeaponResistance.cs [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+		[Description("Resistant to nonmagical Weapons")]
+		Nonmagical = 1,
+		[Description("Resistant to nonmagical weapons that aren't silvered")]
+		NonmagicalSilvered = 2,
+		[Description("Resistant to nonmagical weapons that aren't adamantine")]
+		NonmagicalAdamantine = 3,
+		[Description("Resistant to nonmagical weapons that aren't cold-forged iron")]
+		NonmagicalColdForgedIron = 4,
+		[Description("Resistant to magic weapons")]
+		Magical = 5,
```
Comment (lorddusk) [inline FantasyModuleParser/NPC/Models/Action/Enums/WeaponResistance.cs:18]: Consistency error

File: FantasyModuleParser/NPC/Models/Action/Enums/WeaponImmunity.cs [hunk 7]
```diff
@@ -6,12 +6,16 @@ public enum WeaponImmunity
...
 		Nonmagical = 1,
+		
 		[Description("Immune to nonmagical weapons that aren't silvered")]
 		NonmagicalSilvered = 2,
+		
 		[Description("Immune to nonmagical weapons that aren't adamantine")]
 		NonmagicalAdamantine = 3,
+		
 		[Description("Immune to nonmagical weapons that aren't cold-forged iron")]
 		NonmagicalColdForgedIron = 4,
```
Comment (lorddusk) [inline FantasyModuleParser/NPC/Models/Action/Enums/WeaponImmunity.cs]: Inconsistency errors in the writing of Weapons/weapons.

File: FantasyModuleParser/NPC/Models/Action/Enums/WeaponResistance.cs [hunk 8]
```diff
@@ -6,14 +6,19 @@ public enum WeaponResistance
...
 		[Description("Resistant to nonmagical weapons that aren't silvered")]
 		NonmagicalSilvered = 2,
+		
 		[Description("Resistant to nonmagical weapons that aren't adamantine")]
 		NonmagicalAdamantine = 3,
+		
 		[Description("Resistant to nonmagical weapons that aren't cold-forged iron")]
 		NonmagicalColdForgedIron = 4,
+		
 		[Description("Resistant to magic weapons")]
```
Comment (lorddusk) [inline FantasyModuleParser/NPC/Models/Action/Enums/WeaponResistance.cs]: Inconsistency errors in the writing of Weapons/weapons.

**Comment by EngineerSuite2** [comment]
All changes have been made as requested

**Pull request merged**